crates/astria-composer/src/executor/mod.rs

```diff
@@ -345,15 +363,17 @@ impl Executor {
                             new_nonce = new_nonce,
                             "successfully submitted transction bundle"
                         );
+
                         nonce = new_nonce;
                         bundles_drained += 1;
                     }
                     Err(error) => {
-                        error!(bundle = %telemetry::display::json(&SizedBundleReport(&bundle)),
-                                %error, "failed submitting bundle to sequencer during shutdown; \
-                                    aborting shutdown");
+                        error!(bundle =  %telemetry::display::json(&SizedBundleReport(&bundle)),
```

Comment:
Manual format becasue Rustfmt seems to fail:
```suggestion
                        error!(
                            bundle =  %telemetry::display::json(&SizedBundleReport(&bundle)),
                            %error,
                            "failed submitting bundle to sequencer during shutdown; \
                             aborting shutdown",
                        );
```